shard.yml

```diff
@@ -12,7 +12,7 @@ targets:
 
 crystal: '>= 1.0.0'
 
-dependencies:
```

Comment:
I don't think we should make this change. won't library not work standalone now?

Reply:
Yeah, need to leave these.

Reply:
Yes this was a mistake. Intended to make this change on `kernel_introspection` and made it on the wrong repo 😃